security/firely-auth/firely-auth-deploy.rst

```diff
@@ -159,7 +165,7 @@ Use of the SQL Server user store requires Microsoft SQL Server version 2016 or n
 Using your favorite database administration tool:
 
 - create a new database, e.g. 'firely_auth_store'
-- in this database, execute the script ``scripts/InitializeSchema.sql``, available in the binaries
+- in this database, execute the scripts from the ``scripts/SqlServer/`` folder, available in the binaries, or let the application run the migrations by itself (but then the user must have enough privileges).
```

Comment:
Can we summarise the permissions needed here?

Reply:
done